rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -20,6 +20,15 @@
   synchronize:
     src={{ maas_plugin_dir }} dest=/usr/lib/rackspace-monitoring-agent/plugins/ delete=yes
 
+- name: Drop in wrapper script to run maas plugins in venv
+  template:
+    src: "run_plugin_in_venv.sh.j2"
+    dest: "/usr/lib/rackspace-monitoring-agent/plugins/run_plugin_in_venv.sh"
+    owner: "root"
+    group: "root"
+    mode: "0645"
```

Comment:
So the better question is: Who (which user) runs this script? If it's not root, they don't need the ability to execute it. If it's not someone in the same root group, they also don't need to execute it. So that should determine the permissions here.